python/pyarrow/_s3fs.pyx

```diff
@@ -45,6 +45,98 @@ def finalize_s3():
     check_status(CFinalizeS3())
 
 
+cdef class S3ProxyOptions:
+    """
+    Proxy Options for S3
+
+    Parameters
+    ----------
+
+    scheme : str, default ''
+        Proxy scheme. Valid values: 'http' and 'https'.
```

Comment:
Shouldn't the default simply be "http"?

Reply:
I think we want it to remain empty in the default case where we aren't using a proxy. I'll have to verify, but I think otherwise the AWS SDK might try to use a proxy and fail.
Also [this](https://github.com/aws/aws-sdk-cpp/blob/e4223509ba1bcc5807dfb4f1afef01fdc83ad8c2/aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h#L130) doesn't specify a default, unlike [this](https://github.com/aws/aws-sdk-cpp/blob/e4223509ba1bcc5807dfb4f1afef01fdc83ad8c2/aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h#L66).
But since we're getting rid of this class it shouldn't matter.